R/rowClust.R

```diff
@@ -0,0 +1,114 @@
+#' Clusters rowData using hierarchical clustering
```

Comment:
This does not cluster rowData. This clusters the rows of assay data. 

Suggestion: "Cluster assay rows" (no need to mention hclust here on the title row)